packages/cli/templates/js/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "module": "ESNext",
+    "moduleResolution": "bundler",
+    "resolveJsonModule": true,
+    "allowSyntheticDefaultImports": true
+  },
+  "include": ["vite.config.ts"]
+}
```

Comment:
Although we are not using TypeScript in the JS template, we need this file in order to `import { plugins } from '@storyblok/field-plugin/vite'` in the `vite.config.ts`.